metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/models.scala

```diff
@@ -29,21 +29,35 @@ object Purpose {
   implicit val format: RootJsonFormat[Purpose] = jsonFormat4(Purpose.apply)
 }
 
-final case class Descriptor(
+final case class DescriptorRaw(
```

Comment:
It's a bit confusing seeing this model with a different naming compared to other models used here (`Agreement`, `Purpose`).
I suggest to leave this as `Descriptor` and rename the new (I don't know, like `MetricDescriptor`)

Reply:
Good suggestion, thanks!